tests/TestCase.php

```diff
@@ -47,6 +47,7 @@ abstract class TestCase extends OrchestraTestCase
     protected function setUp(): void
     {
         $_ENV['APP_BASE_PATH'] = realpath(__DIR__.'/../workbench/').'/';
+        $_ENV['NIGHTWATCH_TOKEN'] = '12341234';
```

Comment:
Lets put this in the `phpunit.xml` file. The `APP_BASE_PATH` needs to be here because it is runtime calculated.